rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/SWTFixture.java

```diff
@@ -375,6 +382,26 @@ public SWTBotCheckBox getCheckBox(String locator, SWTLocatorStrategy locatorStra
 		throw new IllegalArgumentException("Unkown locatorStrategy: " + locatorStrategy);
 	}
 
+	@FixtureMethod
+	public String containsActiveTextEditorContent(String searchString) {
+		logger.info("Check if the current active editor {} contains {}", bot.activeEditor().getTitle(), searchString);
+		return Boolean.toString(bot.activeEditor().toTextEditor().getText().contains(searchString));
+	}
+
+	@FixtureMethod
+	public void removeLineFromEditor(int lineNumber) {
+		logger.info("Removing line {} from  editor {}.", lineNumber, bot.activeEditor().getTitle());
+		SWTBotEclipseEditor textEditor = bot.activeEditor().toTextEditor();
+		textEditor.selectLine(lineNumber - 1);
+		textEditor.typeText(" ");
+	}
+
+	@FixtureMethod
+	public void saveActiveEditor() {
+		logger.info("Save editor {}", bot.activeEditor().getTitle());
```

Comment:
`bot.activeEditor()` should be extracted into a variable. If the activeEditor changes between this line and the next one - which is possible - this will be difficult to figure out (since one would first trust the log).